k8s/plugin/Cargo.toml

```diff
@@ -37,7 +37,7 @@ async-trait = "0.1.64"
 serde = "1.0.152"
 serde_json = "1.0.93"
 url = "2.3.1"
-
+ansi_term = "0.12.1"
```

Comment:
should we use `nu-ansi-term` instead since we already depend on it for tracing to avoid using 2 different crates for the same thing?